coldfront/core/project/views.py

```diff
@@ -1075,57 +1075,84 @@ def post(self, request, *args, **kwargs):
         project_user_pk = self.kwargs.get('project_user_pk')
 
         if project_obj.status.name not in ['Active', 'New', ]:
-            messages.error(
-                request, 'You cannot update a user in an archived project.')
-            return HttpResponseRedirect(reverse('project-user-detail', kwargs={'pk': project_user_pk}))
+            messages.error(request, 'You cannot update a user in an archived project.')
+            return HttpResponseRedirect(reverse('project-user-detail', kwargs={'pk': project_obj.pk}))
```

Comment:
The reverse should be to `project-detail`.

Reply:
good catch!